x-pack/plugins/streams_app/public/components/stream_detail_management/index.tsx

```diff
@@ -66,7 +68,11 @@ export function StreamDetailManagement({
     },
     schemaEditor: {
       content: (
-        <StreamDetailSchemaEditor definition={definition} refreshDefinition={refreshDefinition} />
+        <StreamDetailSchemaEditor
```

Comment:
It's just because of this in `index`:

```
if (!definition?.managed) {
    return (
      <RedirectTo
        path="/{key}/management/{subtab}"
        params={{ path: { key, subtab: 'overview' } }}
      />
    );
  }
```

it needs a `_disable` and `_enable`. I won't change anything because this isn't public yet, and the expectation is that `managed` will exist on the stream definitions.

Reply:
Ah, I see, thanks - I will open a separate PR to fix